opentreemap/importer/tasks.py

```diff
@@ -87,18 +87,30 @@ def run_import_event_validation(import_type, import_event_id, file_obj):
             pass
         return
 
-    ie.status = GenericImportEvent.VERIFIYING
+    ie.status = GenericImportEvent.PREPARING_VERIFICATION
     ie.save()
 
-    row_set = ie.rows()
-    validation_tasks = (_validate_rows.subtask(row_set[i:(i+BLOCK_SIZE)])
-                        for i in xrange(0, ie.row_count, BLOCK_SIZE))
+    try:
+        row_set = ie.rows()
+        validation_tasks = (_validate_rows.subtask(row_set[i:(i+BLOCK_SIZE)])
+                            for i in xrange(0, ie.row_count, BLOCK_SIZE))
 
-    final_task = _finalize_validation.si(import_type, import_event_id)
-    res = chord(validation_tasks, final_task).delay()
+        final_task = _finalize_validation.si(import_type, import_event_id)
+        raise Exception('Testing failure')
+        res = chord(validation_tasks, final_task).delay()
 
-    ie.task_id = res.id
-    ie.save()
+        ie.status = GenericImportEvent.VERIFIYING
+        ie.task_id = res.id
+        ie.save()
+    except Exception as e:
+        try:
```

Comment:
this inner try block is pretty ambiguous. It makes sense that the outer try is a generic catch-all because it dumps to a generic error, but can this one be more specific?

I can't tell which statement you were expecting to catch on. The only one that looks like it might fail is `ie.append_error(errors.GENERIC_ERROR, data=[str(e)])`. If so, consider isolating that in the try block, and/or catching something more descriptive and/or adding comments.